x/staging/posts/client/cli/cli_test.go

```diff
@@ -386,6 +393,54 @@ func (s *IntegrationTestSuite) TestCmdQueryRegisteredReactions() {
 						"https://example.com/reaction.jpg",
 						"4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e",
 					),
+					types.NewRegisteredReaction(
+						"cosmos1s3nh6tafl4amaxkke9kdejhp09lk93g9ev39r4",
+						":smile-jpg:",
+						"https://smile.jpg",
+						"5e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e",
+					),
+				},
+				Pagination: &query.PageResponse{
+					NextKey: nil,
+					Total:   0,
+				},
+			},
+		},
+		{
+			name:      "data with subspace is returned properly",
+			args:      []string{"4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e", fmt.Sprintf("--%s=json", tmcli.OutputFlag)},
+			expectErr: false,
+			expectedOutput: types.QueryRegisteredReactionsResponse{
+				RegisteredReactions: []types.RegisteredReaction{
+					types.NewRegisteredReaction(
+						"cosmos1lhhkerae9cu3fa442vt50t32grlajun5lmrv3g",
+						":reaction:",
+						"https://example.com/reaction.jpg",
+						"4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e",
+					),
+				},
+				Pagination: &query.PageResponse{
+					NextKey: nil,
+					Total:   0,
+				},
+			},
+		},
+		{
+			name:      "data with pagination is returned properly",
+			args:      []string{fmt.Sprintf("--%s=%d", flags.FlagLimit, 1), fmt.Sprintf("--%s=json", tmcli.OutputFlag)},
```

Comment:
@dadamu I've tried to execute this and it fails because the returned reaction was only the :smile-jpg: one.
Can you please take a look an fix this? It should be sufficient to edit the expected output as the pagination should return the :smile-jpg: reaction

Reply:
@bragaz After I test with cli_test, it returns the error that EOF. It seems some problem happens on `s.Require().NoError(cfg.Codec.UnmarshalJSON(genesisState[types.ModuleName], &postsData))` in the 61th line.

Is there something I lost to set up?

Reply:
Okay, I fix it with setup app.go file.